06_puppy/patrickmarabeas/main.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+func main() {
+
```

Comment:
I find it a little strange seeing a blank main program. I know the description of the lab doesn't state what you need to include in the main, so you have technically ticked all the boxes, but I feel it makes sense to maybe create a puppy or something to give context to the work being done. Happy to hear your thoughts otherwise.